From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matti Viljanen <matti.viljanen@jolla.com>
Date: Wed, 8 Oct 2025 22:27:13 +0300
Subject: [PATCH] Increase build reproducibility

OBS seems to set SOURCE_DATE_EPOCH already, so let's put it into use.
---
 macros.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/macros.in b/macros.in
index 934e25148..412c10a2f 100644
--- a/macros.in
+++ b/macros.in
@@ -236,12 +236,12 @@ Supplements:   (%{name} = %{version}-%{release} and langpacks-%{1})\
 #	If true, make sure that buildtime in built rpms
 #	is set to the value of SOURCE_DATE_EPOCH.
 #	Is ignored when SOURCE_DATE_EPOCH is not set.
-%use_source_date_epoch_as_buildtime 0
+%use_source_date_epoch_as_buildtime 1
 
 #	If true, make sure that timestamps in built rpms
 #	are not later than the value of SOURCE_DATE_EPOCH.
 #	Is ignored when SOURCE_DATE_EPOCH is not set.
-%clamp_mtime_to_source_date_epoch 0
+%clamp_mtime_to_source_date_epoch 1
 
 #	If enabled, dilute user() and group() requires into recommends
 %_use_weak_usergroup_deps 1
